app/views/pets/show.html.erb

```diff
@@ -12,6 +12,10 @@
       <% end %>
     <p><%= link_to "Edit #{@pet.name}", "/pets/#{@pet.id}/edit" %></p>
     <p><%= link_to "Delete #{@pet.name}", "/pets/#{@pet.id}", method: :delete %></p>
-    <p><%= button_to "Add #{@pet.name} to Favorites", "/favorites/#{@pet.id}", method: :patch %></p>
+    <% if favorites.included_in_favorites?(@pet.id) %>
```

Comment:
I really like this conditional statement! I couldn't figure out how to make this work - Well done!